app/api.py

```diff
@@ -162,7 +161,7 @@ async def create_price(
                     status_code=status.HTTP_403_FORBIDDEN,
                     detail="Proof does not belong to current user",
                 )
-    db_price = crud.create_price(db, price=price, user=current_user)
+    db_price = crud.create_price(db, price=price.model_dump(), user=current_user)
```

Comment:
Price is supposed to be a `PriceCreate`, no?

Reply:
on the crud side we have `Price(**price, user=user)` so I need to check if would work without the `model_dump`

Reply:
If I remove the `model_dump`, I get the following error
```
TypeError: app.models.Price() argument after ** must be a mapping, not PriceCreate
```

Reply:
I moved back the `model_dump` inside the crud file :+1: